src/app/client/src/app/modules/questionnaire/components/input-type-attachment/input-type-attachment.component.ts

```diff
@@ -66,4 +71,37 @@ export class InputTypeAttachmentComponent implements OnInit {
   openFile(file) {
     window.open(file.sourcePath, "_blank");
   }
+
+  async onAddApproval(file){
+    let metaData=this.observationUtil.getAlertMetaData();
+    metaData.type="upload evidence";
+    metaData.size="tiny";
+    metaData.content.title=this.resourceService.frmelmnts.alert.uploadevidencetitle;
+    metaData.content.body.type="checkbox";
+    let html =
+    `
+    ${this.resourceService.frmelmnts.alert.evidence_content_policy}<a href='https://diksha.gov.in/term-of-use.html' target="_blank">${this.resourceService.frmelmnts.alert.evidence_content_policy_label}</a> .${this.resourceService.frmelmnts.alert.uploadevidencecontent}
```

Comment:
dont hardcode the url like this.. please check in the portal how teams and condition URLs are fetched